app/global/MainModule.java

```diff
@@ -96,10 +100,13 @@ protected void configure() {
         bind(HealthProvider.class)
                 .toProvider(ConfigTypedProvider.provider("http.healthProvider.type"))
                 .in(Scopes.SINGLETON);
-        bind(ActorRef.class)
-                .annotatedWith(Names.named("JvmMetricsCollector"))
-                .toProvider(JvmMetricsCollectorProvider.class)
-                .asEagerSingleton();
+
+        // Databases
+        bind(EbeanServer.class)
```

Comment:
That's a good question, frankly I don't know what the correct way to express this is since the actual instance is not created by the provider but instead looked up from `EBean`. Afaik caching the provided instance should be okay, and afaik ebean should always provide the same reference although I have nothing other than a gut feeling to back that up. It's really a question of whether we want to lookup the instance everytime we inject it, or if we want Guice to cache it for us.  Probably won't break either way.